client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -307,9 +315,54 @@ class ThemesMagicSearchCard extends React.Component {
 				onKeyDown={ this.onKeyDown }
 				onClick={ this.onClick }
 				overlayStyling={ this.searchTokens }
-				fitsContainer={ this.props.isBreakpointActive && this.state.searchIsOpen }
 				hideClose={ true }
-			/>
+			>
+				{ renderSuggestions && (
+					<KeyedSuggestions
+						ref={ this.setSuggestionsRefs( 'suggestions' ) }
+						terms={ this.props.filters }
+						input={ this.state.editedSearchElement }
+						suggest={ this.suggest }
+					/>
+				) }
+				{ this.state.searchInput !== '' && (
+					<div className="themes-magic-search-card__icon">
+						<Gridicon
+							icon="cross"
+							className="themes-magic-search-card__icon-close"
+							tabIndex="0"
+							onClick={ this.clearSearch }
+							aria-controls={ 'search-component-magic-search' }
+							aria-label={ translate( 'Clear Search' ) }
+						/>
+					</div>
+				) }
+				{ config.isEnabled( 'theme/showcase-revamp' ) && (
+					<div>
+						<Button
+							onClick={ this.togglePopover }
+							className="components-button themes-magic-search-card__advanced-toggle"
```

Comment:
Do we need `components-button` here? Shouldn't it be part of the component's classNames list already?

Reply:
It does seem to be needed, here's what it looks like with it removed:
![2021-07-14_15-34](https://user-images.githubusercontent.com/937354/125689290-557b1484-158d-4b5e-84e1-12e9bb5498d1.png)
Not exactly sure what's going on, @blackjackkent might have more insight

Reply:
Yeah you can have a button without the component styling (as a link for instance) and it doesn't seem to be applied by default. I'll double check this tomorrow though.